build.sbt

```diff
@@ -4,8 +4,20 @@ lazy val scala213               = "2.13.7"
 lazy val scala212               = "2.12.15"
 lazy val supportedScalaVersions = List(scala213, scala212)
 
-organization := "com.sky"
-name         := "kafka-topic-loader"
+name                   := "kafka-topic-loader"
+organization           := "uk.sky"
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository     := "https://s01.oss.sonatype.org/service/local"
+homepage               := Some(url("https://github.com/sky-uk/kafka-topic-loader"))
+licenses               := List("BSD New" -> url("https://opensource.org/licenses/BSD-3-Clause"))
+developers             := List(
+  Developer(
+    "BiBCD",
+    "BiBCD",
+    "bibcd-jenkins-github-user@skyglobal.onmicrosoft.com",
```

Comment:
this feels like it should come from the environment somehow (not sure how that works with Travis though). and i believe it using the BiBCD user is just an implementation detail, so i'm wondering if this should be like:
```
  Developer(
    "Sky UK OSS",
    "Sky UK OSS",
    "SONATYPE_EMAIL",
Attach files by dragging & dropping, selecting or pasting them.
    url("https://github.com/sky-uk/kafka-topic-loader")
  )
```

Reply:
Yeah sounds like a good idea, getting it from the environment would give us an option so we can just put the repo url (not an email) as a fallback

Reply:
[`824601a` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/824601a84f321bbd7b6f7982316bab2eef9764e8)